checkout/src/checkout.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  CHECKOUT_ENDPOINT_PROD,
+  CHECKOUT_ENDPOINT_SBX,
+  CHECKOUT_LOADED_EVENT,
+} from './config.js';
+
+export default class Checkout extends EventTarget {
+  protected checkoutId: string;
+
+  protected containerId: string;
+
+  protected sandbox: boolean;
+
+  constructor({
+    checkoutId,
```

Comment:
We might want to accept some options (stylings) as well?

Reply:
Yea, that's correct, but that's out of the scope of this ticket.